TAC/Meetings/2024/2024-06-13/CCC-TAC-Minutes-2024-06-13

```diff
@@ -0,0 +1,114 @@
+* Nvidia - Fritz Alder +
+* Red Hat - Yash Mankad + / Ram Pai +
+* TikTok - Mingshen Sun   / Yao Zhang
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Dionna Glaze (Google)
+* Harold Gilkey (AMD)
+* Hugo Vincent (Arm Limited)
+* John Manferdelli
+* Kanth Ghatraju (Oracle)
+* Manu Foutaine (Hushmesh)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Ram Pai (IBM)
+* Ye Li
+
+* Mike Bursell (LF)
+* Riaan Kleinhans (LF)
+* Ben Sternthal (BS)
+
+
+## Welcome
+
+Dan Middleton (DM) opened the call at 7:09 am and reviewed the antitrust policy. 
+
+## Agenda 
+
+DM reviewed the agenda for the meeting. 
+
+## Roll Call / Attendance
+
+Since no votes would be occurring in the meeting, attendance was skipped. 
+
+## Welcome New Community Members
+
+*  Hugo Vincent from ARM introduced themselves.
+
+## Announcements
+
+Riaan Kleinhans (RK) provided an update on the brand repositioning working group, and pointed interested people towards a slack channel. DM stated that he had spoken to Kate and the outreach committee about this workgroup, and they noted it would benefit from TAC representation.
+
+DM called attention to the recent NIST publication. 
+
+
+## CC Summit Feedback
+
+DM asked for feedback on the recent CC summit conference. Ofir Azoulay-Rozanes (OAR) stated that it was the first conference he attended where people understood confidential computing. OAR mentioned that AI was a major topic of discussion, but he felt it was important for people to understand that confidential computing is not just for AI—it is important for any application with sensitive data.
+Fritz Alder said he found the discussion balance between AI solutions to come and current solutions that are available.
+
+Mike Bursell (MB) agreed that the conference over-rotated on AI, he noted there was a lot of interest in business models for attestation and that this should be a focus area for CCC.
+
+It was agreed that DM and Ben Sternthal would work together on helping the SIGs work towards concrete outcomes. 
+
+Ram Pai (RP) stated that the ‘shared workspace’ concept was particularly interesting. 
+
+
+## Tech Topic
+
+John Manferdelli (JM) presented on quantum resistance and confidential computing and his experience implementing quantum-resistant cryptography. [ presentation](./QuantumResistantCCC.pdf)
+Question and answer session and discussion ensured.
+
+## Attestation business models
+
+During the meeting, MB addressed the feedback received from the community regarding potential business models associated with attestation. He suggested the establishment of a working group within the CCC to comprehensively examine both the technical and commercial facets of attestation. Discussions ensued, centered on the topic of attestation. DM expressed concern that creating a new work stream could potentially dilute the efforts of the TAC. MB acknowledged this concern and indicated that the subject matter should be elevated to the GB level, given its broader relevance.
```

Comment:
Suggested rewording:

MB acknowledged this concern and suggested that the issue be escalated to the GB level due to its broader relevance.

Reply:
Updated